easybuild/easyconfigs/b/bam-readcount/bam-readcount-0.7.4-foss-2015b.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Adam Huffman
+# The Francis Crick Institute
+easyblock = 'CMakeMake'
+
+name = 'bam-readcount'
+version = '0.7.4'
+
+homepage = 'https://github.com/genome/bam-readcount'
+description = """Count DNA sequence reads in BAM files"""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/genome/%(name)s/archive']
+
+patches = ['%(name)s-%(version)s-cmake_install_prefix.patch']
+
+builddependencies = [
+    ('SAMtools', '1.3'),
```

Comment:
is listing this as a build dep correct?

also, it seems like `zlib` is missing as a direct dep (cfr. build issue on CentOS 6)